### ISSUE TITLE
[Bug]: Plays music disc 11 (C418) alongside custom disc

### ISSUE THREAD

**Issue by MonstaDash** [spartacus04/jext-reborn#98]
### Describe the bug

When playing some discs it would play disc 11 with it, but when shuffling through other discs and returning to the same disc that would play disc 11 with it would play normally. If it is any help the 13th disc had the glitch

### Steps to reproduce

1. Play disc 13 on the disc.json file
2. It plays the track and music disc 11 (C418) at the same time
3. Shuffle through other randomly chosen discs
4. Disc 13 plays normally

### Expected behaviour

For disc 11 to not play alongside the custom discs

### Server Version

1.19

### Server Type

Spigot

### Discs.json

```json
[
  {
    "title": "Coconut Mall",
    "author": "Shiho Fujii, Atsuko Asahi Ryo Nagamatsu, Yasuaki Iwata",
    "duration": 136,
    "disc-namespace": "music_disc.coconutmallshihofujiiatsukoasahiryonagamatsuyasuakiiwatazero",
    "model-data": 1,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [],
    "fragment-loot-tables": [
      "chests/underwater_ruin_big",
      "chests/underwater_ruin_small"
    ]
  },
  {
    "title": "Route 47",
    "author": "Takuto Kitsuta",
    "duration": 74,
    "disc-namespace": "music_disc.routefourseventakutokitsutaone",
    "model-data": 2,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/village/village_plains_house",
      "chests/village/village_savanna_house",
      "chests/village/village_desert_house",
      "chests/village/village_snowy_house",
      "chests/village/village_taiga_house"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Tears Of Father ~Overseas Edition~",
    "author": "Hidenori Shoji",
    "duration": 275,
    "disc-namespace": "music_disc.tearsoffatheroverseaseditionhidenorishojitwo",
    "model-data": 3,
    "creeper-drop": false,
    "lores": [],
    "loot-tables": [
      "chests/end_city_treasure"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "A Spartan Rises",
    "author": "Martin O'Donnell, Michael Salvatori, Paul Lipson, Lennie Moore, Tom Salta, Brian Trifon, Brian Lee White",
    "duration": 131,
    "disc-namespace": "music_disc.aspartanrisesmartinodonnellmichaelsalvatoripaullipsonlenniemooretomsaltabriantrifonbrianleewhitethree",
    "model-data": 4,
    "creeper-drop": false,
    "lores": [],
    "loot-tables": [
      "chests/stronghold_corridor",
      "chests/stronghold_crossing"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Masked and Wild: D.D.D.",
    "author": "Hirokazu Ando, Yuuta Ogasawara, Jun Ishikawa, Yuki Shimooka",
    "duration": 82,
    "disc-namespace": "music_disc.maskedandwilddddhirokazuandoyuutaogasawarajunishikawayukishimookafour",
    "model-data": 5,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/desert_pyramid"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "The Battle of Blizzard Bridge",
    "author": "Hirokazu Ando, Yuuta Ogasawara, Jun Ishikawa, Yuki Shimooka",
    "duration": 664,
    "disc-namespace": "music_disc.thebattleofblizzardbridgehirokazuandoyuutaogasawarajunishikawayukishimookafive",
    "model-data": 6,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/igloo_chest",
      "chests/village/village_taiga_house"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "It Has To Be This Way",
    "author": "Jimmy Gnecco",
    "duration": 176,
    "disc-namespace": "music_disc.ithastobethiswayjimmygneccosix",
    "model-data": 7,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/ancient_city",
      "chests/ancient_city_ice_box"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Break Off",
    "author": "Yuri Fukuda",
    "duration": 291,
    "disc-namespace": "music_disc.breakoffyurifukudaseven",
    "model-data": 8,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/bastion_bridge",
      "chests/bastion_hoglin_treasure",
      "chests/bastion_other",
      "chests/bastion_treasure",
      "chests/ruined_portal",
      "chests/nether_bridge"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Knight of the Wind",
    "author": "Crush 40",
    "duration": 232,
    "disc-namespace": "music_disc.knightofthewindcrushfourzeroeight",
    "model-data": 9,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [],
    "fragment-loot-tables": []
  },
  {
    "title": "Watch Me Fly...",
    "author": "Crush 40",
    "duration": 312,
    "disc-namespace": "music_disc.watchmeflycrushfourzeronine",
    "model-data": 10,
    "creeper-drop": false,
    "lores": [],
    "loot-tables": [
      "chests/end_city_treasure"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Banana Jungle",
    "author": "Disc Author",
    "duration": 157,
    "disc-namespace": "music_disc.bananajunglediscauthoronezero",
    "model-data": 11,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/jungle_temple"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Lake",
    "author": "Shota Kageyama",
    "duration": 146,
    "disc-namespace": "music_disc.lakeshotakageyamaoneone",
    "model-data": 12,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/shipwreck_map",
      "chests/shipwreck_supply",
      "chests/shipwreck_treasure",
      "chests/village/village_plains_house",
      "chests/village/village_savanna_house",
      "chests/village/village_desert_house",
      "chests/village/village_taiga_house"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Alabaster Icelands 1 (Alternate Ver.)",
    "author": "Go Ichinose, Hitomi Sato, Hiromitsu Maeba & GAME FREAK",
    "duration": 158,
    "disc-namespace": "music_disc.alabastericelandsonealternatevergoichinosehitomisatohiromitsumaebagamefreakonetwo",
    "model-data": 13,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/igloo_chest",
      "chests/village/village_taiga_house"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Menu - Super Smash Bros. Melee (Brawl)",
    "author": "Motoi Sakuraba",
    "duration": 208,
    "disc-namespace": "music_disc.menusupersmashbrosmeleebrawlmotoisakurabaonethree",
    "model-data": 14,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [],
    "fragment-loot-tables": []
  },
  {
    "title": "Let the Battles Begin! - Battle Medley",
    "author": "Shotaro Shima",
    "duration": 198,
    "disc-namespace": "music_disc.letthebattlesbeginbattlemedleyshotaroshimaonefour",
    "model-data": 15,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [],
    "fragment-loot-tables": []
  },
  {
    "title": "Arise in Valor",
    "author": "Martin O'Donnell, Michael Salvatori, Paul Lipson, Lennie Moore, Tom Salta, Brian Trifon, Brian Lee White",
    "duration": 190,
    "disc-namespace": "music_disc.ariseinvalormartinodonnellmichaelsalvatoripaullipsonlenniemooretomsaltabriantrifonbrianleewhiteonefive",
    "model-data": 16,
    "creeper-drop": false,
    "lores": [],
    "loot-tables": [
      "chests/woodland_mansion",
      "chests/pillager_outpost"
    ],
    "fragment-loot-tables": []
  },
  {
    "title": "Wreckage",
    "author": "Kazuma Jinnouchi",
    "duration": 204,
    "disc-namespace": "music_disc.wreckagekazumajinnouchionesix",
    "model-data": 17,
    "creeper-drop": true,
    "lores": [],
    "loot-tables": [
      "chests/bastion_bridge",
      "chests/bastion_hoglin_treasure",
      "chests/bastion_other",
      "chests/bastion_treasure",
      "chests/ancient_city",
      "chests/ancient_city_ice_box",
      "chests/abandoned_mineshaft",
      "chests/stronghold_corridor",
      "chests/stronghold_crossing",
      "chests/underwater_ruin_big",
      "chests/underwater_ruin_small",
      "chests/ruined_portal"
    ],
    "fragment-loot-tables": []
  }
]
```

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.
- [X] I agree to the fact that if this issue becomes inactive it will be closed.

**Comment by Limon4ikXDorORIG** [comment]
I have the same error!
1.19.4

**Comment by spartacus04** [comment]
Hello, thanks for the bug report. Does this happen only with disc 13 or are there any other discs that stop randomly?

**Comment by MonstaDash** [comment]
It seems to only happen with custom discs

**Comment by spartacus04** [comment]
Hi, I was able to fix the bug in [7841ee3](https://github.com/spartacus04/jext-reborn/commit/7841ee38a32d3e2f53b51c5dd94f7907aed5f68a), it should be implemented in the next update
